packages/db/tests/search/test-AsyncOperationSearch.js

```diff
@@ -0,0 +1,331 @@
+'use strict';
+
+const test = require('ava');
+const cryptoRandomString = require('crypto-random-string');
+const { v4: uuidv4 } = require('uuid');
+const omit = require('lodash/omit');
+const range = require('lodash/range');
+const { AsyncOperationSearch } = require('../../dist/search/AsyncOperationSearch');
+const {
+  translatePostgresAsyncOperationToApiAsyncOperation,
+} = require('../../dist/translate/async_operations');
+
+const {
+  destroyLocalTestDb,
+  generateLocalTestDb,
+  fakeAsyncOperationRecordFactory,
+  migrationDir,
+  AsyncOperationPgModel,
+} = require('../../dist');
+
+const testDbName = `asyncOperation_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+
+  t.context.asyncOperationPgModel = new AsyncOperationPgModel();
+  t.context.asyncOperations = [];
+  t.context.asyncOperationSearchTmestamp = 1579352700000;
+
+  range(100).map((num) => (
+    t.context.asyncOperations.push(fakeAsyncOperationRecordFactory({
+      cumulus_id: num,
+      updated_at: new Date(t.context.asyncOperationSearchTmestamp + (num % 2)),
+      operation_type: num % 2 === 0 ? 'Bulk Granules' : 'Data Migration',
+      task_arn: num % 2 === 0 ? cryptoRandomString({ length: 3 }) : undefined,
+    }))
+  ));
+
+  await t.context.asyncOperationPgModel.insert(
+    t.context.knex,
+    t.context.asyncOperations
+  );
+});
+
+test.after.always(async (t) => {
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test('AsyncOperationSearch returns 10 async operations by default', async (t) => {
+  const { knex } = t.context;
+  const dbSearch = new AsyncOperationSearch({});
+  const results = await dbSearch.query(knex);
```

Comment:
super minor nit (with change below) 
```suggestion
  const { results, meta } = await dbSearch.query(knex);
```